Add link to new Node-RED dashboard project. (#1135)
flowforge/website: src/blog/2023/03/comparing-node-red-dashboards.md

<!--more-->

> **UPDATE:** A new project has been started on the next generation of Node-RED Dashboard. You can get the pre-release code on the [githut repo](https://github.com/flowforge/flowforge-nr-dashboard)

Reviewer: Can we instead link to dashboard.FlowFuse.com?

Reviewer: ```suggestion
**UPDATE:** A new project has been started on the next generation of Node-RED Dashboard. You can get the read more about it on the [documentation site](https://dashboard.flowfuse.com)
```

### Dashboard - third place

> **UPDATE:** A new project has been started on the next generation of Node-RED Dashboard. You can get the pre-release code on the [githut repo](https://github.com/flowforge/flowforge-nr-dashboard)

Reviewer: ```suggestion
**UPDATE:** A new project has been started on the next generation of Node-RED Dashboard. You can get the read more about it on the [documentation site](https://dashboard.flowfuse.com)
```
